tests/snappy_pipeline/workflows/test_workflows_cbioportal_export.py

```diff
@@ -0,0 +1,599 @@
+# -*- coding: utf-8 -*-
+"""Tests for the cbioportal_export workflow module code"""
+
+from pathlib import Path
+import textwrap
+
+import pytest
+from ruamel import yaml
+from snakemake.io import Wildcards
+
+from snappy_pipeline.workflows.cbioportal_export import cbioportalExportWorkflow
+
+from .conftest import patch_module_fs
+
+
+@pytest.fixture
+def cbioportal_zscores_get_df_content():
+    """Return contents of Z-Score data frame."""
+    p1 = "P001-T1"
+    p2 = "P002-T2"
+    som_base = (
+        "SOM_CNV_CALLING/output/bwa.copywriter.{p}-DNA1-WGS1/out/"
+        "bwa.copywriter.{p}-DNA1-WGS1_gene_call.txt"
```

Comment:
The ngs_mapping tool & cnv calling tool should be taken from the configuration, rather than hard-coded.